### DISCUSSION TITLE
[Projects Beta] Labels for draft issue

### DISCUSSION THREAD

**Discussion by osmanis** [community/community#8328] [Projects and Issues]
We use separate views for each major feature we're working on. These views are filtered primarily by label. We often use draft issues quite a bit as well. Whenever we create draft issues, they don't show up in our views because we are unable to add a label to them. 

<img src="https://user-images.githubusercontent.com/43759819/144362018-21617ec9-4df2-42d3-99ba-7aa4354aac72.png" width="300" />

The workaround would be to add an additional column in Projects Beta and add a filter condition for it on the view. But that's more issue management that we're trying to avoid.

Label support for draft issues would help here.

**Comment by rileybroughten** [comment]
Hi @osmanis, thank you for your feedback! We are always looking for ways to improve so I will take this back to the team 👍

**Reply by mahmoudmoravej** [reply]
It is a big problem that makes drafts useless in different views. Any update on the progress or any decision for this request?

**Comment by dwainm** [comment]
I'd really like to see drafts become indistinguishable from issues. I'd like them to have all the data and functionality of real issues, just like draft PR's.

**Comment by amarkpayne** [comment]
Just want to bump this again as well. Same problem as above, we filter our boards by labels, but there are items that we want to track in this view that are better suited to draft issues.

**Comment by peterkos** [comment]
bump on this too!

**Comment by kgKevGomez** [comment]
bump on this too!

**Comment by jeesim2** [comment]
bump on this too!

**Comment by vitaly-ps** [comment]
Bump!

This is a show-stopper for us now, making us consider abandoning the GitHub service altogether. Until we do, any update on this issue is more than welcome!

**Comment by Ctibor1974** [comment]
Ahoj

**Comment by Zechst** [comment]
bump on this too!

**Comment by DaveSanders** [comment]
I would argue that both labels milestones should also be able to be created at the project level.  When you have a platform with multiple, interrelated repos, its very difficult to plan the teams future work without having to add things to repos.  In some cases, we need to plan a larger future task, which might affect multiple repos WITHOUT going into the design depth needed to create the dozens of issues.  Sure, we can create a draft, but its really hard to plan when I don't have labels or milestones.

Project issues need to be as feature full as repo issues.

**Comment by Malix-Labs** [comment]
See #108378 that would close this issue

**Comment by justin-robinson** [comment]
Please add this. There are so many usability issues with Github projects. I'm going move us to something else when we get time.

**Comment by baronfel** [comment]
Many teams in the dotnet org would love this feature - we often use Projects to coordinate actions that happen both in GitHub and in internal, private feeds like Azure DevOps - these can be tracked more easily with Draft issues (since they often contain links to resources that can only be accessed internally). It sucks to have to use prefixes/put magic keywords in Draft issues to make them searchable when Label classification is _right there_.

**Comment by mattunleashed** [comment]
Bumping this. I like to use Projects to track ideas and love that I can create them as Drafts via the Github CLI but I want to be able to tag/label and interact with them without having to move them to an existing repository first.
